src/frontend/pages/game.tsx

```diff
@@ -0,0 +1,205 @@
+import Head from 'next/head';
+import styles from '../styles/Game.module.css';
+import Slider from 'react-input-slider';
+import Modal from 'react-modal';
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import React, { ReactElement, useEffect, useRef, useState } from 'react';
+import Image from 'next/image';
+
+const MAX_TAKE_VAL = 10;
+const INIT_TIME_LEFT = 10;
+
+export default function Home(): ReactElement {
+  const [takeVal, setTakeVal] = useState(MAX_TAKE_VAL);
+  const [timeLeft, setTimeLeft] = useState(INIT_TIME_LEFT);
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+
+  useEffect(() => {
+    const interval = setInterval(
+      () => setTimeLeft((timeLeft) => timeLeft - 1),
+      1000,
+    );
+
+    return () => clearInterval(interval);
+  }, [timeLeft]);
+
+  if (timeLeft === 0) {
+    setTakeVal(Math.floor(Math.random() * 11));
+    setTimeLeft(10);
+  }
+
+  const inputOnChange = (eventVal: string) => {
+    const intVal = parseInt(eventVal);
+
+    // TODO: You can't type a '-', not sure we need this first condition
+    if (intVal < 0) {
+      alert('Input cannot be negative');
+      setTakeVal(0);
+    } else if (intVal > 10) {
+      alert('Input cannot be greater than 10');
+      setTakeVal(takeVal);
+    } else {
+      setTakeVal(intVal);
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>RDG NU | Game Page</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.main}>
+        <div className={styles.infoSection}>
+          <p className={styles.infoSectionTitle}>Game</p>
+          <Image
+            src={'/help-icon.svg'}
+            alt={'Help icon'}
+            width={35}
+            height={35}
+            onClick={() => setModalIsOpen(true)}
+          />
+          <Modal
+            isOpen={modalIsOpen}
+            onRequestClose={() => setModalIsOpen(false)}
+            style={{
+              content: {
+                top: '20%',
+                left: '60%',
+                right: 'auto',
+                bottom: 'auto',
+                marginRight: '-50%',
+                transform: 'translate(-50%, -50%)',
+              },
```

Comment:
this isnt a blocker for merging, but we should extrapolate these style dictionaries to a different file. We can't use the CSS file since they're for that component, but I have a couple of these two so for readability it'll be helpful